model/BUILD.bazel

```diff
@@ -6,6 +6,7 @@ The proto_library() rule creates a proto file library to be used in multiple lan
 The java_lite_proto_library() rule takes in a proto_library target and generates java code.
 '''
 load("//model:src/main/proto/format_import_proto_library.bzl", "format_import_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
```

Comment:
Should we also be importing ``java_lite_proto_library`` from somewhere vs. using the native one?

Reply:
The error was that we needed `javalite_toolchain` (since `java_lite_proto_library` implicitly depends on it), which we get from `rules_proto`. This load actually isn't needed, it appears to build without it, but we still need the `http_archive` of rules_proto it in WORKSPACE. I'll remove this load but I think `java_lite_proto_library` exists already? Still unfamiliar with how to read these BUILD files

Reply:
Sorry to clarify, I think we _should_ be loading both java_lite_proto_library and proto_library since it's recommended that we use the Starlark versions of these rather than the native (built-in) versions. I was asking if we could load the Java ones that we're using in addition to proto_library--I didn't intend for you to remove the proto_library load.

(note that it probably should build in either case since the Starlark rules are supposed to be interoperable--this is more of a long-term correctness/best practice situation).

Reply:
Ah I see, in that case we need rules_java as java_lite_proto_library is there, while rules_proto only has proto_library
I just pushed this change

Reply:
Aha, thanks!